foundation/org.eclipse.persistence.core/src/main/java/org/eclipse/persistence/internal/databaseaccess/DatasourceCall.java

```diff
@@ -484,6 +485,25 @@ public void translateCustomQuery() {
         }
     }
 
+    /**
+     * INTERNAL:
+     * Remove line comments from query string.
+     * @param queryString
+     * @return Query string without line comments.
+     */
+    private String removeLineComments(String queryString) {
+        StringTokenizer multiLineTokenizer = new StringTokenizer(queryString, "\n");
```

Comment:
will this work for windows line endings (`\r\n`) as well?

Reply:
I think, that it is tested at NamedNativeQueryJUnitTest.java  line 175